pyvista/core/filters/data_set.py

```diff
@@ -1991,6 +1992,17 @@ def glyph(
         if isinstance(orient, str):
             dataset.active_vectors_name = orient
             orient = True
+        elif isinstance(orient, bool) and orient:
+            try:
+                pyvista.set_default_active_vectors(self)
+            except MissingDataError:
+                warnings.warn("No vector-like data to use for orient. orient will be set to False.")
+                orient = False
+            except AmbiguousDataError as err:
+                warnings.warn(
+                    f"{err}\nIt is unclear which one to use. orient will be set to False."
+                )
+                orient = False
```

Comment:
And in this case, i.e. `orient=True` and there are too many potential vectors to use, I might even consider raising. This has to be a user error, and we have to force the user to do something (either disable orient, or choose vectors to orient by).